contrib/remote-ssh-build.sh

```diff
@@ -34,6 +34,7 @@
 # zfs change-key \
 #     -o keylocation=file:///etc/zfs/keys/rpool.key \
 #     -o keyformat=passphrase rpool
+# # add /etc/zfs/keys/rpool.key to the FILES array in /etc/mkinitcpio.conf
```

Comment:
```suggestion
# sed -i '/FILES=/a FILES+=(/etc/zfs/keys/rpool.key) /etc/mkinitcpio.conf
```
Keeps this in the same spirit as other commands and allows people to copy and paste if they want.